performance-tests/src/scenario-02/s02.test.ts

```diff
@@ -0,0 +1,130 @@
+import { Httpx } from "https://jslib.k6.io/httpx/0.1.0/index.js";
+import { check } from "k6";
+import exec from "k6/execution";
+import { Response } from "k6/http";
+import { ConstantArrivalRateScenario, Options } from "k6/options";
+import { CreateIdentityResponse, IdentityWithToken } from "../models/identity";
+import { StartSyncRunRequestBody, StartSyncRunResponse, SyncRunType } from "../models/sync-run";
+import { createIdentity, exchangeToken } from "../libs/backbone-client/identity";
+
+export const options: Options = {
+    scenarios: {
+        constantRequestRate: {
+            executor: "constant-arrival-rate",
+            rate: 10,
+            timeUnit: "1s",
+            duration: "1m",
+            preAllocatedVUs: 10
+        }
+    }
+};
+
+const client = new Httpx({
+    baseURL: "http://localhost:8081/api/v1/",
+    timeout: 20000 // 20s timeout
+}) as any; // TODO: remove cast to any
```

Comment:
I told you fix the typings and remove the TODO comment afterwards. None of both has happened